.github/copilot-instructions.md

```diff
@@ -1128,8 +1128,15 @@ async def init_integration(
 ) -> MockConfigEntry:
     """Set up the integration for testing."""
     mock_config_entry.add_to_hass(hass)
-    await hass.config_entries.async_setup(mock_config_entry.entry_id)
-    await hass.async_block_till_done()
+    
+    context = nullcontext()
+    if platform := getattr(request, "param", None):
```

Comment:
I'd move the platforms parameter to a separate fixture so we don't need this logic.

Example:
https://github.com/home-assistant/core/blob/1d2599184b49e53450b168c9c45627ac641adaa4/tests/components/zwave_js/conftest.py#L945-L948

https://github.com/home-assistant/core/blob/1d2599184b49e53450b168c9c45627ac641adaa4/tests/components/zwave_js/conftest.py#L923-L938

Reply:
Nice! Yeah I like that better. Updated!

Now I have another PR to update https://github.com/home-assistant/core/pull/151172 😄